res/linux/org.mixxx.Mixxx.metainfo.xml

```diff
@@ -86,7 +86,7 @@
     </screenshot>
   </screenshots>
   <url type="homepage">https://mixxx.org</url>
-  <url type="bugtracker">https://github.com/mixxxdj/mixxx/issues/</url>
+  <url type="bugtracker">https://bugs.launchpad.net/mixxx</url>
```

Comment:
wrong url for bug tracker

Reply:
@daschuer wrong git rerere record? : )

Reply:
it seems like I have messed up the merge. I will do it again.